translations/fr.json

```diff
@@ -0,0 +1,224 @@
+        "Fee": "Frais",
+        "Flea Market": "Marché aux puces",
+        "Flea Price (avg)": "IDK",
+        "Flea Price (low)": "IDK",
+        "Flea Price (optimal)": "IDK",
+        "Found no results for \"{{searchString}}\"": "Pas de résultats pour la recherche \"{{searchString}}\"",
+        "for": "pour",
+        "Format HH:MM:SS": "Format HH:MM:SS",
+        "Frag": "IDK",
+        "Get the full data from {{url}}": "Obtenez les données complètes à partir de {{url}}",
+        "Good Luck!": "Bonne Chance !",
+        "Health": "Vie",
+        "Help for /": "IDK",
+        "Hideout": "Refuge",
+        "Hideout Management skill set to {{managementLevel}}.": "IDK",
+        "I have been online for": "Je suis en ligne depuis",
+        "Intelligence Center": "IDK",
+        "Invalid token": "IDK",
+        "Item Tier": "Rang d'Objet",
+        "kg": "kg",
+        "Latest EFT Changes": "Dernières modifications d'EFT",
+        "Last Updated": "Dernière mise à jour",
+        "Level": "Niveau",
+        "level": "niveau",
+        "LL": "IDK",
+        "Loot Tiers": "Rang de butin",
+        "Loot tiers are divided primarily by the per-slot value of the item": "IDK",
+        "Map": "Carte",
+        "Map made by {{author}}": "Carte réalisée par {{author}}",
+        "minutes": "minutes",
+        "more": "plus",
+        "Name": "Nom",
+        "Need Help or Have Questions?": "Besoin d'aide ou des questions?",
+        "never": "jamais",
+        "Next update": "Prochaine mise à jour",
+        "No matching hideout station found.": "Aucune station de refuge correspondante n'a été trouvée.",
+        "No matching trader found.": "Aucun commerçant correspondant n'a été trouvé.",
+        "No prices available.": "Pas de prix disponibles..",
+        "Note: Progress synced via [TarkovTracker](https://tarkovtracker.io/settings/) will overwrite your hideout settings. \nUse `/progress unlink` to stop syncing from TarkovTracker.": 
+        "IDK",
+        "Net": "IDK",
+        "Pen": "IDK",
+        "Players": "Joueurs",
+        "Please visit {{url}} for more information": "Pour plus d'informations, veuillez consulter {{url}}",
+        "PMC level set to {{level}}.": "IDK",
+        "Price": "Prix",
+        "Price and Item Details": "Prix et détails des objets",
+        "Raid Start": "Début du raid",
+        "Restock alert channel disabled for this server.": "Canal d'alerte du réapprovisionnement désactivé pour ce serveur.",
+        "Restock alert channel set to #{{channelName}}.": "Alerte du réapprovisionnement définie sur #{{channelName}}. ",
+        "Restock alert disabled for {{traderName}}.": "Alerte de réapprovisionnement désactivée pour {{traderName}}.",
+        "Restock alert enabled for {{traderName}}.": "Alerte de réapprovisionnement activée pour {{traderName}}.",
+        "Required Tools": "Outils requis",
+        "seconds": "secondes",
+        "Sell to": "Vendu à",
+        "Skills": "Compétences",
+        "slot": "IDK",
```

Comment:
"slot" in the context of a "space" that an item takes up. Think "slot in a vest" where a slot could be one "space" taken up out of 10.

Example: This vest has 10 "slots" so it can hold 10 items